File: src/ServerPulse.Frontend/src/app/modules/shared/domain/dtos/statistics/response/statistics/loadAmountStatisticsResponse.ts
import { BaseStatisticsResponse } from "../../../../../index";

export interface LoadAmountStatisticsResponse extends BaseStatisticsResponse {
    amountOfEvents: number;
    dateFrom: Date;
    dateTo: Date;
}

export function transformLoadAmountStatisticsResponseDate(response: LoadAmountStatisticsResponse): LoadAmountStatisticsResponse {
    return {
        ...response,
        dateFrom: new Date(response.dateFrom),
        dateTo: new Date(response.dateTo)
    };
}

File: src/ServerPulse.Backend/AnalyzerApi.Core/Dtos/Responses/Statistics/LoadAmountStatisticsResponse.cs
namespace AnalyzerApi.Core.Dtos.Responses.Statistics
{
    public class LoadAmountStatisticsResponse : BaseStatisticsResponse
    {
        public int AmountOfEvents { get; set; }
        public DateTime DateFrom { get; set; }
        public DateTime DateTo { get; set; }
    }
}
